Pipeline_example/scripts/cp_parallel.py

```diff
@@ -0,0 +1,136 @@
+
+
+"""
```

Comment:
Recommend moving this docstring to the first line of the file.

Reply:
Changed! Thanks!